### ISSUE TITLE
Refactor to make game screen look good on all screen sizes

### ISSUE THREAD

**Issue by Zambrella** [Flutter-Buddies/tic_tac_no#16]

**Comment by Zambrella** [comment]
Working on this today

**Comment by Zambrella** [comment]
<img width="595" alt="Screenshot 2021-02-05 at 10 15 28" src="https://user-images.githubusercontent.com/53582591/107020690-1384e380-679b-11eb-842f-a1390f8f3240.png">
Broken on 720p phones

**Comment by slovnicki** [comment]
Hm... I guess we should manually compute the size of table cells, but based on device height, not width

**Comment by Zambrella** [comment]
I'm going to have a play but I'm thinking of having the top section be the dynamically sizing part of the screen.

**Comment by slovnicki** [comment]
Yeah, that in fact sounds better. It would be a little inconvenient to have a small phone AND even smaller grid. Let it always be as wide as it can. That surely leaves _some_ height to be worked with for top section

**Comment by Zambrella** [comment]
![Screenshot_1612537336](https://user-images.githubusercontent.com/53582591/107050520-3b883d00-67c3-11eb-9f3a-20cb4cbf08ab.png)
![Screenshot_1612537334](https://user-images.githubusercontent.com/53582591/107050522-3cb96a00-67c3-11eb-986b-85df267ceb07.png)
![Screenshot_1612537333](https://user-images.githubusercontent.com/53582591/107050523-3cb96a00-67c3-11eb-80ed-2107f773ed24.png)

Got it working on 720p, 1080p and 1440p extended height displays. Not the most fool proof but I think it should cover most cases.

If you're wondering I used a combination of `LayoutBuilder` and `AspectRatio` widgets (along with `Flexible`/`Expanded`).

I have tested both the 'playing state' and the 'won' state.
